tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.ts

```diff
@@ -0,0 +1,316 @@
+import {PolymerElement, html} from '@polymer/polymer';
+import {LegacyElementMixin} from '@polymer/polymer/lib/legacy/legacy-element-mixin';
+import {customElement, property, computed} from '@polymer/decorators';
+import '@polymer/paper-button';
+import '@polymer/paper-dialog';
+import '@polymer/paper-input';
+import '@polymer/paper-slider';
+import * as _ from 'lodash';
+
+// TODO: fix imports when merging.
+import * as tf_backend from '../../../components/tf_backend';
+import * as categorization_utils from '../../../components/tf_categorization_utils/categorizationUtils';
+import {TfImageLoader} from './tf-image-loader';
+import {TagInfo} from '../../../components/tf_utils/utils';
+import '../../../components/tf_categorization_utils/tf-tag-filterer';
+import '../../../components/tf-dashboard-common/tf-dashboard-layout';
+import '../../../components/tf-paginated-view/tf-category-paginated-view';
+import '../../../components/tf_runs_selector/tf-runs-selector';
+
+type RunTagInfo = {
```

Comment:
you might be able to grab the type from category util if you don't want to repeat this type too much.

Reply:
Done, as it's shared with other plugins like histogram.